TA_dataset/bin/dataset_common.py

```diff
@@ -212,7 +279,36 @@ def get_acct_info(self, logger, account=None):
     return acct_dict
 
 
-def get_token(self, account, rw, logger):
+def get_tenant_value(conf, logger):
+    tenant_value = conf.tenant
+    tenant_value = tenant_value.strip()
+    logger.info("The provided tenant value in config is {}".format(tenant_value))
+    if tenant_value.lower() == "false" or tenant_value.lower() == "0":
+        return False
+    return True
+
+
+def get_account_ids(conf, logger):
+    account_ids_array = []
+    tenant = False if conf.tenant == "0" else True
+    if tenant:
+        logger.debug("Account configured on global level")
+        return account_ids_array
```

Comment:
imho this statement is redundant.

Reply:
My point was to remove the return statement now whole section

Reply:
I am confused this is not needed anymore as you have deleted it?

Reply:
removed that if tenant condition. we need this method to fetch the account ids. Hope this make sense